x-pack/solutions/observability/plugins/inventory/public/components/entity_enablement/unauthorized_modal.tsx

```diff
@@ -1,100 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import React from 'react';
-import { i18n } from '@kbn/i18n';
-import {
-  EuiButton,
-  EuiConfirmModal,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiIcon,
-  EuiImage,
-  EuiLink,
-  EuiPanel,
-  EuiSpacer,
-  EuiText,
-  EuiTitle,
-  useEuiTheme,
-  COLOR_MODES_STANDARD,
-} from '@elastic/eui';
-import inventoryLight from '../../assets/entities_inventory_light.png';
```

Comment:
Can we remove these since they're not being used?

Reply:
Good catch, thanks! Removed [here](https://github.com/elastic/kibana/pull/205534/commits/2eca834b86722b679b10491e35a6a5a960f7299f) :)